test/actions/katello/repository/update_redhat_repository_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'katello_test_helper'
+
+module Actions
+  describe Katello::Repository::UpdateRedhatRepository do
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+    include FactoryBot::Syntax::Methods
+
+    let(:action_class) { ::Actions::Katello::Repository::UpdateRedhatRepository }
+    let(:refresh_class) { ::Actions::Pulp::Repository::Refresh }
+    let(:repo) { katello_repositories(:fedora_17_x86_64) }
+
+    it 'plans' do
+      action = create_action(action_class)
+
+      repo.content.update_attributes!(content_url: "/foo/bar")
+      repo.product.provider.update_attributes!(repository_url: "http://cdn.com")
+
+      expected_relative_path = repo.generate_repo_path(repo.generate_content_path)
```

Comment:
nit: I see some value in hard coding the actual expected URL strings here, so that if it does change the test will surely fail whereas if actual and expected somehow become nil for example then we'd get a false positive

Reply:
I like your suggestion. But separate tests have been added with hard coded paths to exclusively test ```generate_repo_path``` and ```generate_content_path``` -> https://github.com/Katello/katello/blob/7ce4bb1a5ba2033d31026438619a34bc6d53b310/test/models/repository_test.rb#L472-L494
I dont want this test to fail if the fixture path changed.